docs/hosting/community-edition-features.md

```diff
@@ -0,0 +1,30 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
```

Comment:
Missing a title front matter (but I'm not 100% sure if that's always needed? @Joffcom)

Reply:
It isn't needed but it is good to have as it lets people know where to find the definition.

Reply:
What would need to be added? 😇

Reply:
see other comment :)